test/TestBuilder.cpp

```diff
@@ -197,16 +215,10 @@ TEST_CASE("JsonBuilder string push_back")
 
     SECTION("push_back std::string")
     {
-        auto itr = b.push_back(b.root(), "", std::string{ "ABCDE" });
+        auto itr = b.push_back<std::string_view>(b.root(), "", std::string{ "ABCDE" });
```

Comment:
Note: We no longer support direct push_back of `std::string`, but you can easily get the same capability via an implicit conversion from `std::string` to `std::string_view`.